src/databricks/labs/ucx/cli.py

```diff
@@ -705,7 +705,17 @@ def migrate_acls(
         workspace_contexts = [ctx]
     else:
         workspace_contexts = _get_workspace_contexts(w, a, run_as_collection, **named_parameters)
-    target_catalog, hms_fed = named_parameters.get("target_catalog"), named_parameters.get("hms_fed", False)
+    target_catalog = named_parameters.get("target_catalog")
+    hms_fed = named_parameters.get("hms_fed", False)
+    dry_run = named_parameters.get("dry_run", False)
+    if dry_run:
+        for workspace_context in workspace_contexts:
+            grants = workspace_context.acl_migrator.snapshot()
+        logger.info(
+            f"Dry run completed. Found {len(grants)} grants. The crawled grants can be found in the 'hms_table_access' table. "
```

Comment:
this is incorrect again - you're overwriting `grants` for each workspace.

Reply:
also - add a link to `{self._ws.config.host}/explore/data/hive_metastore/{inventory_database}/inferred_grants`

Reply:
Fixed